src/components/notification/Notifcation.js

```diff
@@ -52,9 +52,9 @@ class Notification extends Component {
   async readAllNotif() {
     try {
       const { token } = this.props;
-      await axios.patch('https://devrepublic-bn-backend.herokuapp.com/api/v1/notifications/all-read', {}, { headers: { token } });
+      await axios.patch('https://devrepublic-bn-backend.herokuapp.com/api/v1/notifications/all-read', {}, { headers: { token: token || localStorage.getItem('token') } });
```

Comment:
Use environment variable instead of hardcoding the `backend URL`.
This should be applicable on all places where dynamic values are hard-coded

Reply:
removed the URL and used environment variable